README_TTN.md

```diff
@@ -103,6 +107,134 @@ Details are available [here](https://blog.squix.org/2017/07/seeeduino-lora-gps-g
 Compiling the OTAA sketch should make the Seeeduino connect to the Gateway and
 send messages to the TTN.
 
+## Receive data from TTN into IoT-LoRaWan Agent
+
+In order to receive data from TTN the IoT-Lorawan Agent should
+subscribe to the applications on the TTN platform.
+
+The code below is the comand, already configured, that we use in order to connect.
+
+Two keys, one for each application are generated in order to
+connect and used as password.
+Remember to remove the security code.
+
+```bash
+curl --location --request POST 'localhost:4042/iot/services' \
+--header 'fiware-service: openiot' \
+--header 'fiware-servicePath: /' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "services": [
+    {
+      "entity_type": "waste",
+      "apikey": "",
+      "resource": "8CF957200005727C",
+      "attributes": [
+        {
+          "object_id": "temperature_1",
+          "name": "temperature",
+          "type": "Number"
+        },
+        {
+          "object_id": "relative_humidity_1",
+          "name": "relative_humidity",
+          "type": "Number"
+          },
+          {
+            "object_id": "digital_in_1",
+            "name": "flame_detected",
+            "type": "Number"
+            },
+            {
+              "object_id": "digital_in_2",
+              "name": "distance",
```

Comment:
https://gitlab.com/orchestracities/iot/-/blob/master/diorama/src/waste/waste.ino#L118

this is a fillingLevel, so you need to compute it as %. you can compute it in arduino or here, if you compute it here, you need a jexl expression in definition: https://iotagent-node-lib.readthedocs.io/en/2.23.0/expressionLanguage.html